addons/binding/org.openhab.binding.cups/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="cups"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+	<bridge-type id="bridge">
+		<label>CUPS server</label>
+		<description>This bridge represents a CUPS printer server.</description>
+		<config-description>
+			<parameter name="host" type="text">
+				<label>Hostname</label>
+				<description>Hostname or IP address of the pulseaudio server</description>
+				<default>localhost</default>
+				<required>true</required>
+			</parameter>
+			<parameter name="port" type="integer">
+				<label>Port</label>
+				<description>Port of the pulseaudio server</description>
+				<default>4712</default>
+				<required>false</required>
+			</parameter>
+			<parameter name="refreshInterval" type="integer">
+				<label>Refresh Interval</label>
+				<description>The refresh interval in ms which is used to poll given pulseaudio server.</description>
+				<default>30000</default>
+				<required>false</required>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<thing-type id="printer">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge" />
+		</supported-bridge-type-refs>
+		<label>Cups Printer Thing</label>
+		<description>A Cups Printer</description>
+
+		<channels>
+			<channel id="jobs" typeId="jobs" />
+			<channel id="waitingJobs" typeId="waitingJobs" />
+			<channel id="doneJobs" typeId="doneJobs" />
+		</channels>
+		<config-description>
+			<parameter name="name" type="text">
+				<label>Name</label>
+				<description>The name of the printer</description>
+				<required>true</required>
```

Comment:
Is the name really mandatory and not the url maybe enough (from which a name could be derived)?